fcrepo-exts/fcrepo-upgrade-utils
# Make upgrade-utils output OCFL v1.1

**JIRA Ticket**: https://fedora-repository.atlassian.net/browse/FCREPO-3915

# What does this Pull Request do?
Upgrades the versions of ocfl-java and fcrepo-storage-ocfl being used. Also updates the expected test objects to use the ocfl 1.1 style.

I hard-coded the Upgrade on Write to `true`, not sure if we want that to be optional?

# How should this be tested?

Run over a Fedora 4 or 5 repository and see that OCFL 1.1 comes out. There were no functional changes so this is mostly a PR of updating test objects.


# Additional Notes:

* Does this change require documentation to be updated? no
* Does this change add any new dependencies? no
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? no
* Could this change impact execution of existing code? no

# Interested parties
Tag (@ mention) interested parties or, if unsure, @fcrepo-exts/committers
